applications/nrf5340_audio/src/bluetooth/le_audio_cis_gateway.c

```diff
@@ -799,6 +814,15 @@ static void discover_sink_cb(struct bt_conn *conn, struct bt_codec *codec, struc
 
 	if (valid_codec_cap_check(headsets[channel_index].sink_codec_cap,
 				  ARRAY_SIZE(headsets[channel_index].sink_codec_cap))) {
+		if (conn == headsets[AUDIO_CH_L].headset_conn) {
+			bt_codec_allocation_set(&lc3_preset_sink.codec,
+						BT_AUDIO_LOCATION_FRONT_LEFT);
+		} else if (conn == headsets[AUDIO_CH_R].headset_conn) {
+			bt_codec_allocation_set(&lc3_preset_sink.codec,
+						BT_AUDIO_LOCATION_FRONT_RIGHT);
+		} else {
+			LOG_ERR("Unknown connection, cannot set allocation");
+		}
```

Comment:
Nitpick if you need to do other changes (you dont have to make a new commit only for this):
Newline here